--- src/routes/(app)/player/+layout.svelte
+++ src/routes/(app)/player/+layout.svelte
@@ -21,6 +21,7 @@
	import ScrollContainer from '$lib/components/ScrollContainer.svelte'
	import Slider from '$lib/components/Slider.svelte'
	import Tabs from '$lib/components/Tabs.svelte'
+	import AnimatedBackground from '$lib/components/player/AnimatedBackground.svelte'
	import TracksListContainer from '$lib/components/tracks/TracksListContainer.svelte'
	import { initPageQueries } from '$lib/db/query/page-query.svelte.js'
	import { formatArtists, getItemLanguage } from '$lib/helpers/utils/text.ts'
@@ -51,7 +52,7 @@
		class={[
			layoutMode === 'both' && 'w-100 2xl:w-[28dvw]',
			layoutMode === 'list' && 'mx-auto w-full',
-			'player-content z-0 grow items-center gap-x-6 overflow-clip bg-secondaryContainerVariant px-2 pb-2',
+			'player-content z-0 grow items-center gap-x-6 overflow-clip bg-transparent px-2 pb-2',
			isCompactVertical && !isCompactHorizontal && 'player-content-horizontal',
		]}
	>
@@ -75,13 +76,13 @@
		/>

		<div class="mt-2 flex w-full flex-col gap-2 [grid-area:controls]">
-			<div class="w-full rounded-2xl bg-surfaceContainerHighest px-4 py-2">
+			<div class="w-full rounded-2xl bg-surfaceContainerHighest/40 backdrop-blur-xl px-4 py-2">
				<Timeline class="w-full" />
			</div>

			<div
				class={[
-					'flex w-full flex-col gap-6 rounded-2xl bg-secondaryContainer px-4 [grid-area:header]',
+					'flex w-full flex-col gap-6 rounded-2xl bg-secondaryContainer/40 backdrop-blur-xl px-4 [grid-area:header]',
					mainStore.volumeSliderEnabled ? 'pt-8 pb-4' : 'py-8',
				]}
			>
@@ -113,7 +114,7 @@
				{/if}
			</div>

-			<div class="flex h-18 w-full shrink-0 items-center rounded-2xl bg-secondaryContainer px-4">
+			<div class="flex h-18 w-full shrink-0 items-center rounded-2xl bg-secondaryContainer/40 backdrop-blur-xl px-4">
				{#if activeTrack}
					<div class="mr-2 min-w-6 text-center text-body-lg tabular-nums">
						{player.activeTrackIndex + 1}
@@ -268,10 +269,12 @@
	</ScrollContainer>
 {/snippet}

+<AnimatedBackground />
+
 <ListDetailsLayout
	id="full-player"
	mode={layoutMode}
	class={[
-		'grow active-view-player:view-name-[pl-card]',
-		layoutMode === 'both' && 'bg-secondaryContainer',
+		'grow active-view-player:view-name-[pl-card] relative z-10',
+		layoutMode === 'both' && 'bg-background/40 backdrop-blur-2xl',
	]}
	list={playerSnippet}
	details={queueSnippet}
